github.yaml

```diff
@@ -58,6 +58,31 @@ systems:
             - name: resource_path
               description: This field is used as the path portion of the API call url
 
+      createRepo:
+        target:
+          system: github
+          function: api
+        parameters:
+          URL: 'https://api.github.com/orgs/{{ .ctx.org }}/repos'
+          method: POST
+          content:
+            name: '{{ .ctx.name }}'
+            private: true
+        export:
+          repoid: '{{ .data.json.id | int | toString }}'
```

Comment:
Do we really need the `| int | toString`? I prefer using `$data.json.id`, this avoids the a go template rendering.

Reply:
I tried that but It came out float for something which is something like 2.7214 10^7

Reply:
repoid:2.33673173e+08

Reply:
I removed the toString part but I think we need to convert it to int because the repoid wasn't an int